bedrock/gigabit/templates/gigabit/gigabit.html

```diff
@@ -134,7 +134,7 @@ <h1 class="large">{{ _('Mozilla Gigabit Community Fund') }}</h1>
             <h2>{{ _('Get in Touch') }}</h2>
             <p>
               {% trans %}
-              Find us on <a href="http://twitter.com/mozillagigabit">Twitter</a> or send us an <a href="mailto:gigabit@mozilla.com">email</a>.
+              Find us on <a href="http://twitter.com/MozGig">Twitter</a> or send us an <a href="mailto:gigabit@mozilla.com">email</a>.
```

Comment:
Ideally these URLs shouldn't be part of the string, but I also don't think this pages is localized so this string change shouldn't have any l10n impact anyway. Mind putting these into variables the same way they're done in `apply.html`?